src/SDK/Common/Instrumentation.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\SDK\Common;
+
+use OpenTelemetry\API\Behavior\LogsMessagesTrait;
+use OpenTelemetry\SDK\Common\Configuration\Configuration;
+use OpenTelemetry\SDK\Common\Configuration\Variables;
+
+class Instrumentation
```

Comment:
I'm thinking whether we need new class for it or if name of it is correct. It of course depends what is planned here in the future, but maybe for now we could just have a function that would be part of configuration, something like
`isInstrumentationDisabled` ?

Reply:
I'll move it into `Sdk` class, where a similar method already exists.